src/dmd/backend/dwarfdbginf.d

```diff
@@ -26,82 +26,90 @@ generates a separate `.dSYM` file. This file can then later be deployed with the
 application if debug info is needed when the application is deployed.
 */
 
-#if !SPP
-#include        <stdio.h>
-#include        <string.h>
-#include        <stdlib.h>
-#include        <sys/types.h>
-#include        <sys/stat.h>
-#include        <fcntl.h>
-#include        <ctype.h>
-
-#if __DMC__ || __linux__
-#include        <malloc.h>
-#endif
-
-#if __linux__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __DragonFly__ || __sun
-#include        <signal.h>
-#include        <unistd.h>
-#include        <errno.h>
-#endif
-
-#include        "cc.h"
-#include        "global.h"
-#include        "code.h"
-#include        "type.h"
-#include        "outbuf.h"
-#include        "filespec.h"
-#include        "cv4.h"
-#include        "cgcv.h"
-#include        "dt.h"
-
-#include        "aa.h"
-
-#if ELFOBJ
-#include        "melf.h"
-#endif
-#if MACHOBJ
-#include        "mach.h"
-#endif
-
-#if ELFOBJ || MACHOBJ
-
-#if MARS
-#include        "mars.h"
-#endif
-
-#include        "dwarf.h"
-#include        "dwarf2.h"
-
-#define null NULL
-typedef unsigned char ubyte;
-typedef unsigned short ushort;
-#define cast
+module dmd.backend.dwarfdbginf;
```

Comment:
What does `dbginf` mean, "debug info"? If that's the case why not spell it out? I hate these cryptic names.

Reply:
I get tired of typing out long names over and over.

Reply:
It's not like you're going to type this name everyday. It's reference in the makefiles, inside the file itself and `dwarf.h` is reference in three places. All of these will most likely never change. It's not like typing "import std.stdio" which one will do everyday.

Reply:
When I'm working on a file, I type it a lot. I've probably typed it 100 times today.

Reply:
Not sure why you would do that. But whatever, I've approved the PR.